### ISSUE TITLE
Broken

### ISSUE THREAD

**Issue by Wandalen** [jakobhellermann/bevy_mod_debugdump#6]
Thanks for the open source, but samples from the readme are broken.

**Comment by jakobhellermann** [comment]
Broken in which way? The only problem I noticed is that they don't disable the `LogPlugin` like the examples do, so you can't do `cargo run | dot -Tsvg` directly, I fixed that. Otherwise I see no problem with them. Are using using `bevy_mod_debugdump` `0.3.0`?

**Comment by Wandalen** [comment]
I meant examples form the [readme file](https://github.com/jakobhellermann/bevy_mod_debugdump#schedule-graph) 
Is that possible them are outdated?

**Comment by jakobhellermann** [comment]
I don't think so. I pasted them in a new file and everything worked as expected. What error message are you getting?
